README.md

```diff
@@ -16,8 +16,7 @@
 
 Paperless-ngx is a document management system that transforms your physical documents into a searchable online archive so you can keep, well, _less paper_.
 
-Paperless-ngx forked from [paperless-ng](https://github.com/jonaswinkler/paperless-ng) to continue the great work and distribute responsibility of supporting and advancing the project among a team of people. [Consider joining us!](#community-support) Discussion of this transition can be found in issues
-[#1599](https://github.com/jonaswinkler/paperless-ng/issues/1599) and [#1632](https://github.com/jonaswinkler/paperless-ng/issues/1632).
+Paperless-ngx is the official successor to the original 'Paperless' & 'Paperless-ng' projects and is designed distribute responsibility of supporting and advancing the project among a team of people. [Consider joining us!](#community-support).
```

Comment:
👍 

Spotted some minor typos.
```suggestion
Paperless-ngx is the official successor to the original 'Paperless' & 'Paperless-ng' projects and is designed to distribute responsibility of supporting and advancing the project among a team of people. [Consider joining us!](#community-support)
```

The docs use the same wording:
https://github.com/paperless-ngx/paperless-ngx/pull/4542/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R50

There, it actually also says "...projects and is designed **to continue the great work** and distribute responsibility..."

Reply:
Thanks, yea fair enough. I've unified the language.

Reply:
Readme looking good now! 👍 

(FYI, I'm lacking permissions to provide reviews on the repo 😉)